examples/webgl_loader_sea3d_hierarchy.html

```diff
@@ -120,6 +116,9 @@
 				renderer.setPixelRatio( window.devicePixelRatio );
 				renderer.setSize( window.innerWidth, window.innerHeight );
 				renderer.setClearColor( 0x333333, 1 );
+				renderer.shadowMap.enabled = true;
+				renderer.shadowMap.type = THREE.PCFSoftShadowMap;
+				renderer.shadowMap.cullFace = THREE.CullFaceBack;
```

Comment:
@sunag 
Yes, I remember having the same problem porting one of the examples and worked around it prefixing with the target uuid. I'm not much an expert when it comes to loaders and all the formats we support or halfway support, so I didn't really know how to best fix the underlying problem.

You OTOH seem to be having a plan, so I'll be grateful if you fixed it. I guess your changes will be sound and so there shouldn't be any problem getting them merged.